fineract-accounting/src/main/java/org/apache/fineract/accounting/producttoaccountmapping/service/ProductToGLAccountMappingReadPlatformServiceImpl.java

```diff
@@ -167,6 +167,10 @@ public Map<String, Object> fetchAccountMappingDetailsForLoanProduct(final Long l
                 } else if (glAccountForLoan.equals(AccrualAccountsForLoan.INCOME_FROM_GOODWILL_CREDIT_PENALTY)) {
                     accountMappingDetails.put(LoanProductAccountingDataParams.INCOME_FROM_GOODWILL_CREDIT_PENALTY.getValue(),
                             gLAccountData);
+                } else if (glAccountForLoan.equals(AccrualAccountsForLoan.DEFERRED_INCOME_LIABILITY)) {
+                    accountMappingDetails.put(LoanProductAccountingDataParams.DEFERRED_INCOME_LIABILITY.getValue(), gLAccountData);
+                } else if (glAccountForLoan.equals(AccrualAccountsForLoan.INCOME_FROM_DEFERRED_INCOME)) {
+                    accountMappingDetails.put(LoanProductAccountingDataParams.INCOME_FROM_DEFERRED_INCOME.getValue(), gLAccountData);
```

Comment:
The mapping name is "INCOME FROM CAPITALIZATION"

Reply:
Renamed it.